esp-hal/src/soc/esp32/psram.rs

```diff
@@ -616,7 +616,7 @@ pub(crate) mod utils {
                     if clk_mode == PsramClkMode::PsramClkModeDclk {
                         spi.sram_drd_cmd()
                             .modify(|_, w| w.cache_sram_usr_rd_cmd_bitlen().bits(15)); // read command length, 2 bytes(1byte for delay),sending in qio mode in
-                                                                                       // cache
+                        // cache
```

Comment:
just out of curiosity: where is the comment from and could we expand on this? (not really necessary for this PR)